src/db/index_knex.ts

```diff
@@ -1,5 +1,11 @@
 import dotenv from 'dotenv';
-import knex from 'knex';
+
+// Work around the incompatibility between Knex typings
+// and the "module": "nodenext" TS config
+// Will be fixed once a new version of Knex with the following PR is released:
+// https://github.com/knex/knex/pull/5659
+import * as knexModule from 'knex';
+const { knex } = knexModule.default;
```

Comment:
A new Knex version fixing this should be released tonight – https://github.com/knex/knex/pull/5659#issuecomment-1742723386 – so no need to merge the PR yet.